submissions/emmanuelEsuong/htmlForm/htmlForm.html

```diff
@@ -10,6 +10,19 @@
 
```

Comment:
Well done! Remove the extra one space between each element and it will be more readable using putting the break elements in a new line.